cross/opencore-amr/Makefile

```diff
@@ -0,0 +1,28 @@
+PKG_NAME = opencore-amr
+PKG_VERS = 0.1.3
+PKG_EXT = tar.gz
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://sourceforge.net/projects/opencore-amr/files/$(PKG_NAME)
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+
+
+DEPENDS =
+
+HOMEPAGE = https://github.com/Distrotech/opencore-amr/
+COMMENT  = OpenCORE Adaptive Multi Rate (AMR) speech codec library implementation.
+LICENSE  = Apache 2
+
+GNU_CONFIGURE = 1
+
+ifeq ($(findstring $(ARCH), $(ARM5_ARCHES)),$(ARCH))
```

Comment:
I believe I've mentioned this before. If, generally speaking, this function works if it's added _before_ the include, then use this format throughout the PR.
Although I doubt that this is the case currently...the arg is not added when 88f6281 is compiled. Having said that, missing the arg also does not trigger an error, which makes me think: what's the added value of setting the arg then?

Anyway, please recheck and correct :)